docs/getting-started/security-ui.asciidoc

```diff
@@ -82,6 +82,26 @@ Expand this section to access the Overview, Detection & Response, Kubernetes, Cl
 [role="screenshot"]
 image::images/dashboards-pg.png[]
 
+[float]
+=== Rules
+
+Expand this section to view the following pages:
+
+* <<rules-ui-management, *Rules*>>: Create and manage rules to monitor suspicious events.
```

Comment:
Only suggesting this edit because users technically see the Detection rules (SIEM) page in the expanded Rules view. It's not until after they click **Detection rules (SIEM)**, that they're brought to a page titled Rules.


<img width="300" alt="Screenshot 2023-09-07 at 9 53 59 AM" src="https://github.com/elastic/security-docs/assets/79124755/91d58c64-e7e8-46b1-b7ea-a027cb37dff4">


```suggestion
* <<rules-ui-management, *Detection rules (SIEM)*>>: Create and manage rules to monitor suspicious events.
```

Reply:
Hmm, I'm not sure I would call **Detection rules (SIEM)** a page – I see it as a navigation item that leads to the **Rules** page. Similar to how we have **Benchmark rules** leading to the **Benchmark Integrations** page (as described [here](https://security-docs_3884.docs-preview.app.elstc.co/guide/en/security/master/benchmark-rules.html)).

Considering that this line is also followed by the screenshot of the **Rules** page, do you think it might be confusing for users if the screenshot said "**Rules**" but the line above it said "**Detection rules (SIEM)**"?